### DISCUSSION TITLE
JavaScript 异步编程

### DISCUSSION THREAD

**Discussion by giscus** [youngjunning/blog.zisheng.pro#128] [Announcements]
# JavaScript 异步编程

围绕异步编程领域，现代软件设计正在加速旋转，就为了让程序在一个时间内做更多的事情。当你使用更新更强大的 API 时，你会发现在更多的情况下，使用异步编程是唯一的途径。以前写异步代码很困难，现在也需要你来适应，但是已经变容易了很多。

https://youngjuning.js.org/60882b7c0e5d/

**Comment by qualvey** [comment]
这个页面用触摸板滚动会很卡
